src/main/java/com/neuvector/Scanner.java

```diff
@@ -305,143 +301,28 @@ private static ScanRepoReportData runScan(String[] cmdArgs, NVScanner nvScanner,
         }else{
-            reportData = parseScanReport(getScanReportPath(nvScanner.getNvMountPath()));
+            String scanResultJson = "scan_result.json";
+            String reportJsonString = outputCollector.substring(outputCollector.toString().indexOf(scanResultJson) + scanResultJson.length());
+            reportData = parseScanReport(reportJsonString);
         }
 
         return reportData;
     }
 
-    private static String getMountPath(NVScanner nvScanner){
-        String mountPath = ":" + CONTAINER_PATH;
-        String nvPath = nvScanner.getNvMountPath();
-        if( nvPath != null && nvPath.length() > 0){
-            if( nvPath.charAt(nvPath.length() - 1) == '/' ){
-                nvPath = nvPath.substring(0, nvPath.length() - 1);
-            }
-            mountPath = nvPath + mountPath;
-        }else {
-            mountPath = CONTAINER_PATH + mountPath;
-        }
-        return mountPath;
-    }
-
-    private static String appendBindMountSharedSuffixIfRequired(final Boolean isBindMountShared, final String mountPath, final Logger log) {
-        String updatedMountPath = mountPath;
-        if(isBindMountShared != null && isBindMountShared){
-            updatedMountPath = new StringBuilder(mountPath).append(":z").toString();
-        }
-        log.info("Using volume binding: {}", updatedMountPath);
-        return updatedMountPath;
-    }
-
-    private static String getScanReportPath(String path){
-        String scanReportPath = "";
-        if(path == null || path.length() == 0){
-            scanReportPath = CONTAINER_PATH + "/" + SCAN_REPORT;
-        }else{
-            scanReportPath = removeLastSlash(path) + "/" + SCAN_REPORT;
-        }
-
-        return scanReportPath;
-    }
-
-    private static String removeLastSlash(String str){
-        if(str != null && str.length() > 0 && str.charAt(str.length() - 1) == '/'){
-            return str.substring(0, str.length() - 1);
-        }else{
-            return str;
-        }
-    }
-
-    private static String generateScannerName(){
-            String SALTCHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
-            StringBuilder salt = new StringBuilder();
-            Random rnd = new Random();
-            while (salt.length() < 6) { // length of the random string.
-                int index = (int) (rnd.nextFloat() * SALTCHARS.length());
-                salt.append(SALTCHARS.charAt(index));
-            }
-            String saltStr = salt.toString();
-            return saltStr;
-    }
-
     private static String maskCredential(String message, String credential){
         return message.replace(credential, "******");
     }
 
-    static String getDockerUserGroupCmdArg(String scanReportPath) {
-        // No user arg if file exists, and it is owned by root
-        Boolean ownedByRoot = ownedByRoot(scanReportPath);
-        if (ownedByRoot != null && ownedByRoot) {
-            return null;
-        }
+    public static String[] masked(String[] runCmd) {
```

Comment:
Very important.. 

We were sadly printing the password in the logs:
```
Running container scan via: podman run --rm --entrypoint /usr/bin/bash -v /var/run/docker.sock:/var/run/docker.sock -e SCANNER_REPOSITORY=fortifydocker/sscdemo -e SCANNER_TAG=23.2.0 -e SCANNER_LICENSE= -e SCANNER_REGISTRY=https://docker-all.repo.sonatype.com -e SCANNER_SCAN_LAYERS=true -e SCANNER_REGISTRY_USERNAME=ktugay -e SCANNER_REGISTRY_PASSWORD=MY-PASSWORD-HERE registry.hub.docker.com/neuvector/scanner:latest -c /usr/local/bin/monitor && echo 'scan_result.json' && cat /var/neuvector/scan_result.json
```

Now it is masked:
```
Running container scan via: podman run --rm --entrypoint /usr/bin/bash -v /var/run/docker.sock:/var/run/docker.sock -e SCANNER_REPOSITORY=fortifydocker/sscdemo -e SCANNER_TAG=23.2.0 -e SCANNER_LICENSE= -e SCANNER_REGISTRY=https://docker-all.repo.sonatype.com -e SCANNER_SCAN_LAYERS=true -e SCANNER_REGISTRY_USERNAME=ktugay -e SCANNER_REGISTRY_PASSWORD=**** registry.hub.docker.com/neuvector/scanner:latest -c /usr/local/bin/monitor && echo 'scan_result.json' && cat /var/neuvector/scan_result.json
```